<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Multiple Particle Tracking</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/resources.qrc">
    <normaloff>:/icons/icons/node-magnifier.png</normaloff>:/icons/icons/node-magnifier.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QTableWidget" name="summary_view">
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="autoScroll">
       <bool>false</bool>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>43</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>RM</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>File name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Trajectories</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Valid trajectories</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QTreeWidget" name="summary_total">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>24</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="autoScroll">
       <bool>false</bool>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="itemsExpandable">
       <bool>false</bool>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="headerMinimumSectionSize">
       <number>43</number>
      </attribute>
      <attribute name="headerDefaultSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="headerStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>total</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>total_trajectories</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>total_valid_trajectories</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionImport_files"/>
    <addaction name="actionExport_files"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <widget class="QMenu" name="menuConfigurations">
     <property name="title">
      <string>Configurations</string>
     </property>
     <addaction name="actionDiffusivity_ranges"/>
     <addaction name="actionApplication_configuration"/>
    </widget>
    <addaction name="actionStart_analysis_tp"/>
    <addaction name="actionStart_analysis"/>
    <addaction name="separator"/>
    <addaction name="menuConfigurations"/>
    <addaction name="separator"/>
    <addaction name="actionRemove_selected"/>
    <addaction name="actionClear_summary"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionImport_files"/>
   <addaction name="actionExport_files"/>
   <addaction name="separator"/>
   <addaction name="actionRemove_selected"/>
   <addaction name="actionClear_summary"/>
   <addaction name="separator"/>
   <addaction name="actionStart_analysis"/>
   <addaction name="actionStart_analysis_tp"/>
   <addaction name="separator"/>
   <addaction name="actionDiffusivity_ranges"/>
   <addaction name="actionApplication_configuration"/>
  </widget>
  <action name="actionImport_files">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/icons/document-import.png</normaloff>:/icons/icons/document-import.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Import files...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionExport_files">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/icons/document-export.png</normaloff>:/icons/icons/document-export.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Export files...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionStart_analysis_tp">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/icons/control-double.png</normaloff>:/icons/icons/control-double.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Start analysis</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionRemove_selected">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/icons/minus.png</normaloff>:/icons/icons/minus.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Remove selected</string>
   </property>
  </action>
  <action name="actionClear_summary">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/icons/broom.png</normaloff>:/icons/icons/broom.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Clear summary</string>
   </property>
  </action>
  <action name="actionDiffusivity_ranges">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/icons/ui-slider-050.png</normaloff>:/icons/icons/ui-slider-050.png</iconset>
   </property>
   <property name="text">
    <string>Diffusivity ranges</string>
   </property>
  </action>
  <action name="actionApplication_configuration">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/icons/application-task.png</normaloff>:/icons/icons/application-task.png</iconset>
   </property>
   <property name="text">
    <string>Application configuration</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/icons/information.png</normaloff>:/icons/icons/information.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionStart_analysis">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/icons/icons/control.png</normaloff>:/icons/icons/control.png</iconset>
   </property>
   <property name="text">
    <string>Start Analysis (without TrackPy)</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
